app/config/tstvhq/rooms.json

```diff
@@ -249,12 +246,12 @@
         {
             "id": "belfry",
             "name": "Belfry",
-            "type": "CHATSTREAM",
+            "type": "JITSI",
             "capacity": 25,
             "muteRoom": false,
             "hideVideo": false,
             "description": "Meet your new best friend, a fruit bat or insect bat from BatWorld! Donate at batworld.org",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/LUPERCALIA/LUPERCALIA-BG-belfry.png",
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/vhq/hexennacht/belfry.png",
             "iframeOptions": {
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
```

Comment:
if this is going to be a `JITSI` room we should get rid of the URL yes?

Reply:
This should be IFRAME, i made mistake. revisin

Reply:
do you mean `CHATSTREAM`?

Reply:
i assume not per other comment